.github/workflows/continuous-integration-workflow.yml

```diff
@@ -43,11 +43,11 @@ jobs:
       - name: Install Python dependencies
         run: poetry install
 
-      - name: Run LISAv3 hello-world
-        run: poetry run python lisa/main.py --debug
-
       - name: Run black/flake8/isort
         run: poetry run flake8
 
       - name: Run mypy static type checking
         run: poetry run mypy --strict --namespace-packages .
+
+      - name: Run LISAv3 hello-world
+        run: poetry run python lisa/main.py --debug
```

Comment:
The hello-world may be slow in future. My point is to run faster part earlier, so it fails earlier. Anyway, I will change the order back, since there are different points.